modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v2/form_data.rb

```diff
@@ -157,6 +157,96 @@ def stamp_text
           "#{last_name.truncate(35)} - #{ssn.last(4)}"
         end
 
+        def claimant_first_name
```

Comment:
Why not use the claimant object directly instead of using this layer of methods? This isn't much different than just putting all the claimant data retrieval on the model, which also felt redundant and is why we wanted the Claimant object in the first place :)